help/en/package/jmri/jmrit/withrottle/Protocol.shtml

```diff
@@ -200,7 +200,7 @@ RL3]\[D&amp;RGW 341}|{3}|{S]\[RGS 41}|{41}|{L]\[Test Loco}|{1234}|{L
       the labels for the different turnout states. For example:</p>
 
       <pre class="code">
-PTT]\[Turnouts}|{Turnout]\[Closed}|{2]\[Thrown}|{4
+PTT]\[Turnouts}|{Turnout]\[Closed}|{2]\[Thrown}|{4]\[Unknown}|{0]\[Inconsistent}|{8
```

Comment:
What is the meaning of this? Closed = 2, Thrown = 4, Unknown = 0, Inconsistent = 8?
It makes sense, since Turnout.CLOSED = 2, Turnout.THROWN = 4, NamedBean.INCONSISTENT = 8.

But NamedBean.UNKNOWN = 1.

Reply:
You are correct on this. Turnouts should treat 1 as UNKNOWN.